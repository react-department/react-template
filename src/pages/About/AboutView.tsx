import { useTranslation } from 'react-i18next';

import type { ReactElement } from 'react';

/**
 * About page view
 */
function AboutView(): ReactElement {
  const { t } = useTranslation();
  return (
    <div>
      <p>{ t('about.pageTitle') }</p>
    </div>
  );
}

export default AboutView;
