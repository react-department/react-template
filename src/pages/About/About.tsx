import AboutView from './AboutView';

import type { ReactElement } from 'react';

function About(): ReactElement {
  return (
    <AboutView />
  );
}

export default About;
