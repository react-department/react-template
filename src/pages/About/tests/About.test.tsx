import { render } from '../../../../__tests__/test-utils';
import About from '../About';

describe('<About />', () => {
  test('should render component', async () => {
    const { asFragment } = render(<About />);
    expect(asFragment()).toMatchSnapshot();
  });
});
