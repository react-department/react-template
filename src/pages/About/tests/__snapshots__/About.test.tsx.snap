// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render component 1`] = `
<DocumentFragment>
  <div>
    <p>
      About
    </p>
  </div>
</DocumentFragment>
`;
